cli/gardener_ci/whdutil.py

```diff
@@ -13,50 +13,87 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import multiprocessing
+import argparse
+import logging
+
+import uvicorn
 
 import ci.util
 
 
-def start_whd(
-    cfg_set_name: str,
-    port: int=5000,
-    production: bool=False,
-    workers: int=4,
-):
-    import whd
-    whd.configure_whd_logging()
+LOGGING_CONFIG = {
+    "version": 1,
```

Comment:
nit: please consistently use either single-quotes or double-quotes (match existing codebase - in case of cc-utils that would be single-quotes)